packages/react-composites/src/composites/CallWithChatComposite/components/AddPeopleDropdown.styles.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IButtonStyles, IContextualMenuStyles, Theme } from '@fluentui/react';
+
+/**
+ * @private
+ */
+export const themedCopyLinkButtonStyles = (mobileView: boolean | undefined, theme: Theme): Partial<IButtonStyles> => ({
```

Comment:
nitty nit: Switch the order of arguments so you don't have to explicitly pass `undefined`: `themedCopyLinkButtonStyles(theme)` vs `themedCopyLinkButtonStyles(undefined, theme)`.

Reply:
updated